app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,358 @@
+  <string name="copying_assets">نسخ الأصول</string> 
+  <string name="failed_create_project_dir">فشل إنشاء مشروع dir!</string> 
+  <string name="failed_write_file">فشل كتابة الملف:٪ s</string> 
+  <string name="asset_copy_failed">فشل نسخ القالب إلى التخزين!</string> 
+  <string name="cannot_create_temp">فشل إنشاء دليل مؤقت للمشروع!</string> 
+  <string name="project_created_successfully">تم إنشاء المشروع بنجاح!</string> 
+  <string name="msg_begin_project_write">بدء عملية الكتابة</string> 
+  <string name="msg_folder_creation_failed">فشل إنشاء مجلد!</string> 
+  <string name="msg_file_creation_failed">فشل إنشاء الملف!</string> 
+  <string name="msg_folder_created">تم إنشاء المجلد بنجاح!</string> 
+  <string name="msg_file_created">تم إنشاء الملف بنجاح!</string> 
+  <string name="msg_folder_exists">المجلد موجود بالفعل!</string> 
+  <string name="msg_file_exists">الملف موجود بالفعل!</string> 
+  <string name="msg_invalid_name">اسم غير صحيح!</string> 
+  <string name="text_create">خلق</string> 
+  <string name="msg_can_contain_slashes">إذا كان المسار يحتوي على فاصل ملفات (\ '/ \') ، فسيتم إنشاء الدلائل (الرئيسية) المطلوبة إذا لم تكن موجودة.</string> 
+  <string name="folder_name">إسم الملف</string> 
+  <string name="file_name">اسم الملف</string> 
+  <string name="new_name">اسم جديد</string> 
+  <string name="new_file">ملف جديد</string> 
+  <string name="create_auto_layout">قم بإنشاء ملف تخطيط تلقائيًا</string> 
+  <string name="new_java_class">فئة Java جديدة</string> 
+  <string name="new_xml_resource">مورد XML جديد</string> 
+  <string name="new_folder">ملف جديد</string> 
+  <string name="title_confirm_delete">تأكيد الحذف</string> 
+  <string name="msg_confirm_delete">هل تريد بالتأكيد حذف: %s</string> 
+  <string name="copied">تم النسخ بنجاح!</string> 
+  <string name="msg_rename_file">أدخل اسمًا جديدًا للملف / المجلد.</string> 
+  <string name="renamed">تمت إعادة التسمية بنجاح!</string> 
+  <string name="rename_failed">غير قادر على إعادة تسمية الملف!</string> 
+  <string name="deleted">حذف بنجاح!</string> 
+  <string name="delete_failed">تعذر حذف الملف!</string> 
+  <string name="copy_path">مسار النسخ</string> 
+  <string name="rename_file">إعادة تسمية</string> 
+  <string name="delete_file">حذف</string> 
+  <string name="file_options">خيارات الملف</string> 
+  <string name="action_closeAll">أغلق الكل</string> 
+  <string name="action_closeOthers">أغلق الآخرين</string> 
+  <string name="action_closeThis">أغلق هذا</string> 
+  <string name="all_saved">تم حفظ جميع الملفات!</string> 
+  <string name="msg_newfile_dest">الأماكن/</string> 
+  <string name="restype_drawable">قابل للرسم</string> 
+  <string name="restype_layout">تَخطِيط</string> 
+  <string name="restype_menu">قائمة الطعام</string> 
+  <string name="restype_other">آخر</string> 
+  <string name="classtype_class">فصل</string> 
+  <string name="classtype_activity">نشاط</string> 
+  <string name="classtype_interface">واجهه المستخدم</string> 
```

Comment:
When translating this with Google translate, it translates to 'User interface'. Could you confirm if this actually says 'User interface' or just 'Interface'?

It should be 'Interface'.

Reply:
Iam an Egyptian and i speak Arabic, and i can confirm that on Arabic it means Interface